src/lang/parser.mly

```diff
@@ -144,7 +144,7 @@ expr:
   | BOOL                             { mk ~pos:$loc (Ground (Bool $1)) }
   | FLOAT                            { mk ~pos:$loc (Ground (Float  $1)) }
   | STRING                           { mk ~pos:$loc (Ground (String $1)) }
-  | VAR                              { mk ~pos:$loc (Var $1) }
```

Comment:
Can this and the one below be moved to the `mk` level?

Reply:
Good idea, I have just implemented this!